collector/info_schema_processlist.go

```diff
@@ -60,104 +61,23 @@ var (
 // Metric descriptors.
 var (
 	processlistCountDesc = prometheus.NewDesc(
-		prometheus.BuildFQName(namespace, informationSchema, "threads"),
-		"The number of threads (connections) split by current state.",
-		[]string{"state"}, nil)
+		prometheus.BuildFQName(namespace, informationSchema, "processlist_total"),
```

Comment:
These are gauges, so they should not end in `_total`. The original metric name here is correct. Maybe `processlist_threads`?